telegram/ext/_picklepersistence.py

```diff
@@ -191,12 +275,11 @@ def _dump_singlefile(self) -> None:
             'callback_data': self.callback_data,
         }
         with self.filepath.open("wb") as file:
-            pickle.dump(data, file)
+            _BotPickler(self.bot, file, protocol=pickle.HIGHEST_PROTOCOL).dump(data)
```

Comment:
curiosity: why the `HIGHEST_PROTOCOL`?

Reply:
Provides performance increase for py3.8+